### DISCUSSION TITLE
Workflow_dispatch not work

### DISCUSSION THREAD

**Discussion by MikiraSora** [community/community#25249] [Actions]
<p>I make a workflow and want to build my program in my repo.<br>
And I define <strong>workflow_dispatch</strong> so that other user could fork and build them directly.<br>
But I notice that Github Actions not raise a new workflow run task when I press button “Run workflow”.<br>
What’s did I wrong?<br>
<img src="https://user-images.githubusercontent.com/7549173/181089851-ad89e3cb-91b5-4b88-845a-0ae21c6d53fe.gif" alt="1" data-base62-sha1="r3FF2Z81rPwJeQkTY2Zxb1Ffzau" width="690" height="348" class="animated"></p>

**Comment by MikiraSora** [comment]
<p>here is my workflow file <a href="https://github.com/MikiraSora/OngekiFumenEditor/edit/master/.github/workflows/BuildProgram.yml" rel="noopener nofollow ugc">link</a></p>

**Comment by gavin-shaw** [comment, marked as answer]
<p>If you are testing this right now (Wed 16 Mar), Github actions is experiencing degraded performance.</p>
<p>Neither my pushes nor dispatches are running, without any changes my site.</p>
<p>Keep an eye on <a href="https://www.githubstatus.com/" rel="noopener nofollow ugc">https://www.githubstatus.com/</a>.</p>
<p>Once Github Actions has a green tick, try again.</p>

**Comment by CJSmith-0141** [comment]
<p>There is an outage, GitHub Actions is degraded. See <a href="https://www.githubstatus.com/" rel="noopener nofollow ugc">https://www.githubstatus.com/</a></p>

**Comment by MikiraSora** [comment]
<p>Thanks to reply and I know :smiley:</p>
